src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/conditionFields.tsx

```diff
@@ -37,41 +39,58 @@ function ConditionFields({
-    <React.Fragment>
+    <Wrapper>
       {conditions.map(({match, category}, index) => {
-        const displayDescription = index === 0;
         const showLegacyBrowsers = category === DynamicSamplingInnerName.LEGACY_BROWSERS;
         return (
-          <Fields key={index}>
-            <SelectField
-              label={displayDescription ? t('Category') : undefined}
-              help={displayDescription ? t('This is a description') : undefined}
-              name={`category-${index}`}
-              value={category}
-              onChange={value => onChange(index, 'category', value)}
-              choices={categoryOptions}
-              inline={false}
-              hideControlState
-              showHelpInTooltip
-              required
-              stacked
-            />
-            <TextField
-              label={displayDescription ? t('Match Conditions') : undefined}
-              help={displayDescription ? t('This is a description') : undefined}
-              placeholder={
-                showLegacyBrowsers ? t('No match condition') : 'ex. 1* or [I3].[0-9].*'
-              }
-              name={`match-${index}`}
-              value={match}
-              onChange={value => onChange(index, 'match', value)}
-              disabled={showLegacyBrowsers}
-              inline={false}
-              hideControlState
-              showHelpInTooltip
-              required
-              stacked
-            />
+          <FieldsWrapper key={index}>
+            <Fields>
+              <SelectField
+                label={t('Category')}
+                help={t('This is a description')}
+                name={`category-${index}`}
+                value={category}
+                onChange={value => onChange(index, 'category', value)}
+                choices={categoryOptions}
+                inline={false}
+                hideControlState
+                showHelpInTooltip
+                required
+                stacked
+              />
+              <StyledField
+                label={t('Match Conditions')}
+                help={t('This is a description')}
+                inline={false}
+                hideControlState
+                showHelpInTooltip
+                flexibleControlStateSize
+                required
+                stacked
+              >
+                <StyledTextareaField
+                  placeholder={
+                    showLegacyBrowsers
+                      ? t('No match condition')
+                      : 'ex. 1* or [I3].[0-9].*'
+                  }
+                  name={`match-${index}`}
+                  value={match}
+                  onChange={value => onChange(index, 'match', value)}
+                  disabled={showLegacyBrowsers}
+                  inline={false}
+                  autosize
+                  hideControlState
+                  stacked
+                />
+                <FieldHelp>{t('Press enter to add a new match condition')}</FieldHelp>
```

Comment:
```suggestion
                <FieldHelp>{t('You can include multiple values by putting each value on a separate line')}</FieldHelp>
```
Not sure if this is better 🤷

Reply:
oh thanks! I was looking for a better phrase